### PR TITLE
Feature/release 1.0.10

### PR DESCRIPTION
**Pull request by markperdue**

### CODE REVIEW

**Review by webdjoe — APPROVED** [review]
Great work, merge if you approve my changes

File: src/pyvesync/vesyncoutlet.py [hunk 1]
```diff
@@ -19,7 +17,8 @@ def __init__(self, details, manager):
         self.details = {}
         self.energy = {}
         self.update_energy_ts = None
-        self._energy_update_interval = ENERGY_UPDATE_INT
+        self._energy_update_interval = manager.energy_update_interval
+        self._energy_update_check = manager.energy_update_check
```
Comment (markperdue) [inline src/pyvesync/vesyncoutlet.py]: Did manager.energy_update_check get left out of VeSync? I don't see it there

Comment (markperdue) [reply src/pyvesync/vesyncoutlet.py]: nm, I see that a future commit removes this

**Comment by markperdue** [comment]
changes look good, thanks! merging

**Pull request merged**